include/SFML/Network/UdpSocket.hpp

```diff
@@ -199,7 +199,7 @@ class SFML_NETWORK_API UdpSocket : public Socket
     ////////////////////////////////////////////////////////////
     // Member data
     ////////////////////////////////////////////////////////////
-    std::vector<char> m_buffer; //!< Temporary buffer holding the received data in Receive(Packet)
+    std::vector<char> m_buffer{std::vector<char>(MaxDatagramSize)}; //!< Temporary buffer holding the received data in Receive(Packet)
```

Comment:
I can't just use `{MaxDatagramSize}` since that tries to constructor a vector of size 1 with a single element of value `MaxDatagramSize`. If we switch from `char` to `std::byte` then this problem is fixed. Hopefully we can start using `std::byte` in more places soon.